File: backend/backend/ENTITY/Entities/Exercise.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace LifeGuideProject.API.ENTITY.Entities
{
    [Table("Exercise", Schema = "public")]
    public class Exercise
    {
        [Key]
        public int id { get; set; }
        public string videoLink { get; set; }
        public int likedCount { get; set; }
        public string dislikedCount { get; set; }
        public int videoDegree { get; set; }
    }
}

File: frontend/src/app/Models/exercise.ts
export class Exercise{
    public id:number;
    public videoLink:string="";
    public likedCount:number;
    public dislikedCount : number;
    public videoDegree : number;
    public videoTitle : string;

    constructor(id:number,videoLink:string,likedCount:number,dislikedCount:number,videoDegree:number, videoTitle : string){
        this.id = id;
        this.videoLink = videoLink;
        this.likedCount = likedCount;
        this.dislikedCount = dislikedCount;
        this.videoDegree = videoDegree;
        this.videoTitle = videoTitle;
    }
}